### ISSUE TITLE
Can you add a new class called "Zend\Db\Sql\Predicate\NotBetween"?

### ISSUE THREAD

**Issue by ntd1712** [zendframework/zend-db#14]
The NotBetween class is basically same as Between class, except its specification value is replaced with '%1$s NOT BETWEEN %2$s AND %3$s'. For example:
```php
<?php
/**
 * Zend Framework (http://framework.zend.com/)
 *
 * @link      http://github.com/zendframework/zf2 for the canonical source repository
 * @copyright Copyright (c) 2005-2015 Zend Technologies USA Inc. (http://www.zend.com)
 * @license   http://framework.zend.com/license/new-bsd New BSD License
 */

namespace Zend\Db\Sql\Predicate;

class NotBetween extends Between
{
    protected $specification = '%1$s NOT BETWEEN %2$s AND %3$s';
}
```
Without NotBetween class, my code will look like this:

```php
$between = new Between($identifier, "'$minValue'", "'$maxValue'");

if ('notBetween' === $predicate) {
     $between->setSpecification('%1$s NOT BETWEEN %2$s AND %3$s');
}

$predicate->addPredicate($between);
```

With NotBetween class, my code will look much more simple:

```php
$predicate->$predicate($identifier, "'$minValue'", "'$maxValue'");
```

Thanks for reading and I hope you may consider this!  =)

**Comment by samsonasik** [comment]
@ntd1712 I created PR #15 for it

**Comment by ntd1712** [comment]
@samsonasik 
How to use it when you forgot to add this method in Zend\Db\Sql\Predicate\Predicate.php? =)

```php

    /**
     * Create "NOT BETWEEN" predicate
     *
     * Utilizes NotBetween predicate
     *
     * @param  string $identifier
     * @param  int|float|string $minValue
     * @param  int|float|string $maxValue
     * @return Predicate
     */
    public function notBetween($identifier, $minValue, $maxValue)
    {
        $this->addPredicate(
            new NotBetween($identifier, $minValue, $maxValue),
            ($this->nextPredicateCombineOperator) ?: $this->defaultCombination
        );
        $this->nextPredicateCombineOperator = null;

        return $this;
    }
```

**Comment by samsonasik** [comment]
@ntd1712 done ;)

**Comment by ntd1712** [comment]
:+1:

**Comment by weierophinney** [comment]
Fixed with #15, which will release with 2.6.0.

**Comment by ntd1712** [comment]
:+1:
